pyqtgraph/graphicsItems/ViewBox/tests/test_ViewBox.py

```diff
@@ -1,53 +1,27 @@
 #import PySide
 import pyqtgraph as pg
+import pytest
 
-app = pg.mkQApp()
-qtest = pg.Qt.QtTest.QTest
+QRectF = None
+app = None
+win = None
+vb = None
 
-def assertMapping(vb, r1, r2):
-    assert vb.mapFromView(r1.topLeft()) == r2.topLeft()
-    assert vb.mapFromView(r1.bottomLeft()) == r2.bottomLeft()
-    assert vb.mapFromView(r1.topRight()) == r2.topRight()
-    assert vb.mapFromView(r1.bottomRight()) == r2.bottomRight()
 
-def test_ViewBox():
-    global app, win, vb
-    QRectF = pg.QtCore.QRectF
-    
-    win = pg.GraphicsWindow()
-    win.ci.layout.setContentsMargins(0,0,0,0)
-    win.resize(200, 200)
-    win.show()
-    vb = win.addViewBox()
-    
-    # set range before viewbox is shown
-    vb.setRange(xRange=[0, 10], yRange=[0, 10], padding=0)
-    
-    # required to make mapFromView work properly.
-    qtest.qWaitForWindowShown(win)
-    
-    g = pg.GridItem()
-    vb.addItem(g)
-    
+def test_resize():
+    # test resize
+    win.resize(400, 400)
     app.processEvents()
```

Comment:
There's an order issue here--previously this was tested before setting the aspect ratio constraint, but now it is after?